scripts/van_allen_tools/migrate_van_allen_workspaces.py

```diff
@@ -131,68 +128,214 @@ def make_create_workspace_request(workspace_name, auth_domains, namespace=NAMESP
     return create_ws_request
 
 
+def copy_workspaces_workflows(destination_workspace_namespace, destination_workspace_name, source_workspace_namespace, source_workspace_name):
+    """Copy Van Allen Lab workspaces Workflows."""
+    # Get the list of all the workflows
+    try:
+        workflow_list = fapi.list_workspace_configs(source_workspace_namespace, source_workspace_name)
+
+        # Store all the workflow names
+        workflow_name_list = []
+
+        for workflow in workflow_list.json():
+            # Get workflow config (overview config)
+            workflow_config = workflow['methodRepoMethod']
+
+            workflow_name_list.append(workflow_config['methodName'])
```

Comment:
you could do 
```workflow_name = workflow['methodRepoMethod']['methodName']
     workflow_name_list.append(workflow_name)```

I think calling it a config and then adding it to a list called workflow_name_list is a bit confusing since config is also something else